server/index-node/src/resolver.rs

```diff
@@ -130,6 +134,17 @@ impl IndexingStatus {
             _ => Ok(None),
         }
     }
+    fn u64_from_value(value: &q::Value, key: &'static str) -> Result<u64, Error> {
```

Comment:
There's a `TryFromValue` trait that we should implement for `u64` instead of introducing `u64_from_value`.